Riot/Modules/Home/AllChats/AllChatsViewController.swift

```diff
@@ -310,14 +310,18 @@ class AllChatsViewController: HomeViewController {
             return super.tableView(tableView, numberOfRowsInSection: section)
         }
         
-        return dataSource.tableView(tableView, numberOfRowsInSection: section)
+        return dataSource?.tableView(tableView, numberOfRowsInSection: section) ?? 0
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard let sectionType = sectionType(forSectionAt: indexPath.section), sectionType == .invites else {
             return super.tableView(tableView, cellForRowAt: indexPath)
         }
         
+        guard let dataSource = dataSource else {
```

Comment:
*Totally unimportant:* would be nice to start using the shortened version :)
```suggestion
        guard let dataSource else {
```

Reply:
I agree though last time I used it the CI was not happy, presumably because it was using some older version of Swift. Do you know if this has changed recently?

Reply:
Should be fine now, CI is using [Xcode 14.2](https://github.com/vector-im/element-ios/blob/0b6aa1f0dadba1cf7c8d85da2ad5819dc92d15b3/fastlane/Fastfile#L24).